Assets/Editor/CustomAttributeDrawers.cs

```diff
@@ -19,7 +19,7 @@ public class ReadOnlyDrawer : PropertyDrawer
 		ReadOnlyAttribute readOnlyAttribute = (ReadOnlyAttribute)attribute;
 
 		// Use a specified name if there is one
-		string displayName = ( readOnlyAttribute.displayName != "" ) ? readOnlyAttribute.displayName : property.name;
+		string displayName = ( readOnlyAttribute.displayName != "" ) ? readOnlyAttribute.displayName : property.displayName;
```

Comment:
The displayName is the "pretty name". I don't know why I didn't see it before. Maybe it's a Unity 5 thing. Maybe I'm dumb. Moral of the story is you don't have to put in a name for ReadOnly if the default name style works.